main.js

```diff
@@ -0,0 +1,58 @@
+let user = "John Doe";
+console.log(user);
+console.log('----------------');
+
+const student = "Maxim";
+console.log(student);
+console.log('----------------');
+
+user = student; // now the value "user" = "Мaxim"
+console.log(user);
+console.log('----------------');
+
+let test = 1;
+test++;
+test += '1'; // now the value "test" = 21
+console.log(test);
+console.log('----------------');
+
+test -= 1; // now the value "test" = 20
+console.log(test);
+console.log('-----------------');
+
+test = !test; //now the value "test" = false
+console.log(test);
+console.log('--------------');
+
+const prim = {
+  mas: [2, 3, 5, 8],
+};
```

Comment:
Is it required to put the massive into the object? Looks like over-engineering. Doesn't make sense here.